content/docs/SUSHI/running/_index.md

```diff
@@ -19,24 +19,84 @@ SUSHI is executed from the command line. The general form of the SUSHI execution
 where options include the following (in any order):
 
 ```text
--o, --out <out>     the path to the output folder
 -d, --debug         output extra debugging information
+-h, --help          output usage information
+-i, --init          initialize a SUSHI project
+-o, --out <out>     the path to the **fsh-generated** folder
 -p, --preprocessed  output FSH produced by preprocessing steps
+--require-latest    indicate that SUSHI should halt if the latest SUSHI version is not installed
 -s, --snapshot      generate snapshot in Structure Definition output (default: false)
--i, --init          initialize a SUSHI project
 -v, --version       print SUSHI version
--h, --help          output usage information
 ```
 
 {{% alert title="Tip" color="success" %}}
-If you run SUSHI from your FSH project directory, and accept the defaults, the command can be shortened to `sushi .`
+If you run SUSHI from your FSH project directory, and accept the defaults, the command can be shortened to `sushi .` or simply `sushi`.
```

Comment:
FYI: We temporarily broke support for running `sushi` by itself (without any arguments).  The fix for that was just merged today, but not yet released.

Reply:
Seems to work for me. Anyway, the doc shouldn't be held up by bugs.